tests/e2escenarios/e2e_java_test.go

```diff
@@ -40,6 +41,10 @@ var _ = Describe("odo java e2e tests", func() {
 	// Test Java
 	Context("odo component creation", func() {
 		It("Should be able to deploy a git repo that contains a wildfly application without wait flag", func() {
+			if runtime.GOARCH == "s390x" {
```

Comment:
I would say create one different context and run all the spec that needs to be skipped on `runtime.GOARCH == "s390x"` and apply condition 

```
if runtime.GOARCH == "s390x" {
				Skip("Skipping test because there is no supported wildfly builder image.")
			}
```
in ginkgo built-in tear down steps `JustBeforeEach`

Reply:
@prietyc123  Thanks. I have finished the change with your comments. Please review again.

Reply:
@zhengxiaomei123 Thanks for addressing the concern 👍

Reply:
@zhengxiaomei123 I believe these tests will run on some machine and not a cluster, unless its a case like our CI.

GOARCH here basically refers to case where this is s390x, not the cluster. Are you assuming that both cluster and where the test will run will both the s390x? Is it really safe to make such an assumption?

I think it will be safer to use an env variable, say CLUSTER_ARCH=s390x, whose default value will be whatever GOARCH is

Reply:
@mohammedzee1000 Thanks for the concern. I think both machine and cluster, the Golang is required to run these tests. Wherever Golang installed, the GOARCH should be s390x.